packages/adapters/src/process-sequence-adapter.ts

```diff
@@ -42,6 +44,7 @@ async function getRunnerConfigForStoredSequence(sequencesRoot: string, id: strin
         author: validPackageJson.author || "",
         keywords: validPackageJson.keywords || [],
         repository: validPackageJson.repository || "",
+        language: language || engines?.node ? "js" : "unknown"
```

Comment:
Patryk, why is this different to kube/docker adapters?

Reply:
i forgot to align changes across sequence adapters. fixed